_posts/blog/2018-10-17-ReviewGarcíaMárqu

```diff
@@ -0,0 +1,63 @@
+ 
+#Gabriel García Márquez. Harry Ransom Center. The University of Texas. Web.16 de Octubre 2018.https:[//hrc.contentdm.oclc.org/digital/collection/p15878coll51/]
+
+
+Así de extraño como el nacimiento del último de los Buendía que nació con una cola de cerdo en la famosa novela 100 años de Soledad del colombiano Gabriel García Márquez, es extraño para muchos de sus paisanos que el archivo privado de este escritor, premio Nobel de literatura 1982, reposen en el Centro de Archivos Harry Ransom Center en la Universidad de Texas en Austin, en los Estados Unidos. País que vetó por muchos años al escritor por la relación de éste con Fidel Castro y por ser catalogado por muchos como comunista.
+
+Pero por qué la familia del escritor querría que sus archivos, los originales de sus novelas más famosas, sus notas, sus memorias, sus fotos y en fin lo que condensa parte su vida privada y que lo hizo famoso públicamente como escritor estuviese tan lejos de su macondiana tierra?...
+
+
+...Uno de sus hijos, Rodrigo García, declaró en una ocasión, que la familia, madre e hijos siempre sintieron el compromiso de hacer que el archivo de su padre llegará a la mayor audiencia posible. En este sentido, el proyecto de digitalizar el archivo por parte de la  reconocida institución Harry Ransom Center parecía lo más cercano a este deseo...
+
+
+...El archivo digitalizado de Gabriel García Márquez abarca más de 27.000 imágenes entre sus documentos y fotografías. Esta colección accesible a todo público mediante la web incluyen manuscritos originales de muchas de sus obras como “100 años de Soledad” y “El Coronel No tiene quién le Escriba”. También una parte de sus memorias escritas en la obra “En Agosto nos Vemos”, que el escritor nunca consideró lista para publicar. El resto de esta obra puede consultarse  en sala. 
+
+En la colección digital se encuentra fotografías que muestran las relaciones del escritor, desde sus encuentros con amigos, también famosos, como Fidel Castro hasta sus relaciones en la intimidad de su familia, como por ejemplo fotos con sus abuelos.
+
+También dentro de la colección, que no está digitalizada,  se pueden consultar la correspondencia del escritor como por ejemplo, las cartas que se escribía con el escritor Plinio Apuleyo Mendoza...
```

Comment:
This comment is confusing to me. Are you reviewing the digital collection, or the entire collection?